### DISCUSSION TITLE
Extending headless-recorder to support other exports

### DISCUSSION THREAD

**Discussion by psiinon** [checkly/puppeteer-recorder#202] [Q&A]
How easy is it to extend headless-record to support other exports, eg a selenium script?
Can that be plugged in or would that require changes to this repo?
If the latter, would you be interested in contributions that did that?

**Comment by ianaya89** [comment]
@psiinon thanks for reaching out. Adding support to other platforms is not in our roadmap, we consider that everything is moving forward with Puppeteer & Playwright. Regardless, we are always open to discussions and evaluate what our users want. 

Adding a new export will require adapting our Code Generator engine. To be honest, I do not know right now how challenging this will be because I do not have experience with Selenium but probably needs some extra effort.

I will dive more into this during the next few days and post updates here.
